x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/Connector.java

```diff
@@ -612,15 +572,11 @@ public int hashCode() {
             serviceType,
             status,
             syncCursor,
-            syncNow
+            syncNow,
+            isDeleted
         );
     }
 
-    @Override
-    public String getWriteableName() {
-        return NAME;
-    }
-
```

Comment:
I can't find reference to usage of this also being deleted in the PR. Just confirming it's safe to delete?

Reply:
Ah I guess if it's an override then there's still a base level fallback?